app/logger/logger.js

```diff
@@ -27,19 +31,43 @@ if (process.env.APPINSIGHTS_CONNECTIONSTRING) {
     })
   )
 } else {
-// if AppInsights is not enabled, send logs to console
-  transportTypes.push(new transports.Console({
-    format: format.combine(
-      sampleResponseBodyData(),
-      redactSensitiveData(),
-      format.align(),
-      format.colorize(),
-      format.printf(info => `${info.level}: ${info.message}${Object.keys(info).length > 2 ? `\n${jsonStringify(info)}` : ''}`)
-    )
-  }))
+  // if AppInsights is not enabled, send logs to console
+  transportTypes.push(
+    new transports.Console({
+      format: format.combine(
+        sampleResponseBodyData(),
+        redactSensitiveData(),
+        format.align(),
+        format.colorize(),
+        format.printf(
+          info =>
+            `${info.level}: ${info.message}${
+              Object.keys(info).length > 2 ? `\n${jsonStringify(info)}` : ''
+            }`
+        )
+      )
+    })
+  )
+}
+
+if (!process.env.EVENT_HUB_DISABLED) {
```

Comment:
have you tried setting to false? often env vars are passes as strings, i.e. "false" == true 🙄